features/product/viewing_products/accessing_product_wich_does_not_exist.feature

```diff
@@ -0,0 +1,14 @@
+@viewing_products
+Feature: Accessing a product which does not exist
+    In order to have a good navigation
+    As a visitor
+    I want to be able to be informed that a product does not exits
+
+    Background:
+        Given the store operates on a single channel in "United States"
+
+    @ui
+    Scenario: Accessing a product which does not exist
+        Given the store has a product "T-shirt banana"
+        When I open page "en_US/products/44 Magnum"
```

Comment:
WDYT about something like `When I try to reach unexistent product`? Behind this step we can just call `$this->showPage->open(['slug' => 'unexisten_product', '_locale' => $localeCode]);`